src/Microsoft.AspNet.TestHost/TestServer.cs

```diff
@@ -14,18 +14,20 @@
 
 namespace Microsoft.AspNet.TestHost
 {
-    public class TestServer : IServerFactory, IDisposable
+    public class TestServer : IDisposable
     {
         private const string DefaultEnvironmentName = "Development";
         private const string ServerName = nameof(TestServer);
-        private static readonly IFeatureCollection ServerInfo = new FeatureCollection();
-        private Func<IFeatureCollection, Task> _appDelegate;
+        private RequestDelegate _appDelegate;
         private IDisposable _appInstance;
         private bool _disposed = false;
+        private IHttpContextAccessor _httpContextAccessor;
```

Comment:
No, IHttpContextAccessor usage was moved into IHttpContextFactory which isn't useful to TestServer since it prefers to create the DefaultHttpContext directly.